docs/products/kafka/kafka-mirrormaker/concepts/disaster-recovery/migration-and-dr-high-level-approach.rst

```diff
@@ -0,0 +1,17 @@
+Migration and Disaster Recovery high level approach
```

Comment:
Where is disaster recovery in this file? Should the title be only about migration?

Reply:
I was referencing to a document which had Migration, active-active, active-passive setups and DR content in it. But since we are splitting these into multiple sections, the DR is included in the active-active and active-passive setup instead of this section. Do you feel the need for this section or does it seem redundant?